docs/_sidebar.md

```diff
@@ -1,11 +1,11 @@
 - [Overview](home)
-- Onboarding
-  - [Public officer](onboard-public-officers-using-non-se-machines)
-    - [WOG Security verification](manage-security-verification-for-wog-account)
-      - [Reset security verification](reset-security-verification-for-wog-account)
-  - [Vendor](onboard-vendors-to-techpass)
-- [Log in to TechPass Portal](log-into-techpass-portal)
-- [Edit Profile](edit-profile)
-- [Access SGTS services](access-sgts-services-using-techpass)
-- [Reset TechPass MFA](reset-techpass-mfa-for-new-device)
+- Getting started
+  - [Public officer onboarding](onboard-public-officers-using-non-se-machines)
```

Comment:
u might want to check if this menu item has too many chars and can be displayed in the navigation side bar.
if too long then can consider:
Public officer onboard
Vendor onboard

Reply:
Done